supplier_app/tests/test_model.py

```diff
@@ -189,4 +190,27 @@ def test_is_token_expired_false(self):
         )
         self.assertFalse(
             companyuniquetoken.is_token_expired
-        )
\ No newline at end of file
+        )
+
+
+class TestTaxpayerComment(TestCase):
+    def setUp(self):
+        self.data = {
+            'taxpayer': TaxPayerFactory(),
+            'user': UserFactory(),
+            'message': 'please upload updated AFIP form',
+        }
+
+        self.comment = TaxpayerComment.objects.create(**self.data)
+
+    def test_taxpayer_comment_persists_in_db(self):
+        self.assertEqual(
+            self.comment,
+            TaxpayerComment.objects.last()
+        )
```

Comment:
You should execute what you are testing here! Here we have just an assertion!

Reply:
solved!